README.md

```diff
@@ -7,67 +7,98 @@ Perses
+
+Perses is going to tackle multiple different goals:
+
+1. It aims to become a **standard** dashboard visualization tool for Prometheus and other datasources. It will focus on
+   being GitOps-compatible and thus enabling a smooth "dashboards as code" workflow via a new and well-defined dashboard
+   definition model.
+2. While becoming another visualization tool, Perses also aims to provide different npm packages, so it can benefit to
+   anyone that would like to integrate data visualization in his UI. For example, these packages might be used to
+   improve the display of the data in the Prometheus UI.
+3. It also aims to offer a Kubernetes-native mode in which dashboard definitions can be deployed into and read from
+   individual application namespaces (Using CRDs)
+4. To be dashboard as code friendly, we want to provide a complete static validation of the dashboard format. That means
+   you will be able to validate your dashboard in a CI/CD totally statically certainly using
+   the Perses CLI (named `percli`)
+5. The architecture should support plugins (at least for the panels)
 
-Perses aims to become a dashboard visualization tool for Prometheus and other datasources. It will focus on being
-GitOps-compatible and thus enabling a smooth "dashboards as code" workflow via a new and well-defined dashboard definition model.
+## Status
 
-It also aims to offer a Kubernetes-native mode in which dashboard definitions can be deployed into and read from individual application namespaces.
+While we already released a certain amount of versions, Perses is still in an early alpha stage and still work in
+progress. The current pieces that are in place are:
+
+* The Plugin architecture has finally reached a stable point.
+    * The plugins concerned the Variables, the Panels, The Queries and the Datasources definitions.
+    * To provide a good static validation, the backend is using multiple Cue schemas and the CLI has the `lint` command.
+      All schemas are available in the [schemas](./schemas) folder.
+* A backend REST API provides R/W access to dashboard and datasource definitions.
+* While the UI is still in progress, we already have:
+    * a beginning of navigation that will help to move from a dashboard to another.
+    * a support of the following panel types:
+        * Time series charts.
+        * Gauge panels.
+        * Stat panels (single value with sparkline).
+        * Mardown panels (as an alternative to the Text panel)
+    * The editing of dashboard is well advanced. Remaining work most of the remaining work is in panel options, right
+      now you have to use the JSON editor for a lot of properties, but those will be form controls soon
 
-## Status
+The dashboard data model is still evolving along with the dashboard implementation and new requirements.
 
-Perses is an early work in progress and is far from production-ready. The current pieces that are in place are:
+## Install
```

Comment:
Should the H2 be something like `## Getting Starting` and then change to H3s for `### Install` and `### Building from source`

Reply:
That feels weird for me to add a `Getting Started` section with just the `Install` section.

Reply:
yeah good point!